bentoml/db.py

```diff
@@ -48,3 +52,24 @@ def create_session(session_maker):
         raise BentoMLException(e)
     finally:
         session.close()
+
+
+def upgrade_db(engine, db_url):
+    # alembic add a lot of import time, so we lazy import
+    from alembic import command
+    from alembic.config import Config
+    from sqlalchemy import inspect
+
+    logger.debug('Updating tables')
```

Comment:
I think this logger is no longer accurate, add something similar to the if/else below?